src/StudyUtils.in.jsm

```diff
@@ -289,6 +461,10 @@ class StudyUtils {
       this._telemetry({study_state: "installed"}, "shield-study");
     }
   }
+
+  /**
+  * @async
+  */
```

Comment:
I hate this method, but it's a core feature of Study Utils.  See "about.md" and "survival"